packages/terra-base/docs/README.md

```diff
@@ -1,20 +1,30 @@
 # Terra Base
 
-The base component receives customized translation messages from the application and passes translations into I18nProvider wrapper and sets minimal global styles for an application.
-Global styles include CSS to help normalize box-sizing, reset margins/paddings, and define global font styles.
+The `terra-base` component will handle locale changes, manage the locale loading state, and receive customized translation messages from an application and pass theme into the `terra-in18n` I18nProvider. It also sets minimal global styles for an application; styles include CSS to help normalize box-sizing, reset margins and paddings, and define global font styles.
+
+**Note: This component should be imported before the other terra components in your app.** This is to help ensure that these styles are defined at the beginning of the [extracted stylesheet](https://github.com/webpack-contrib/extract-text-webpack-plugin) and are inherited down to the other terra components. Additionally, when used to internationalize an application, it will wrap the application’s root component such that the entire application is within the same configured i18n context.
```

Comment:
Should or must?

Reply:
... and are ~~~inherited down to~~~ **inherited by** the other terra components.